VAMobile/ios/Podfile.lock

```diff
@@ -1027,4 +1027,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 71cfabcb52e3b2ad52dad40bf80370529df9a239
 
-COCOAPODS: 1.12.1
+COCOAPODS: 1.11.3
```

Comment:
Looks like the pods and podfile need to be synced with `develop`, this change is specifying an older version than what's on `develop`